src/hotspot/share/opto/generateOptoStub.cpp

```diff
@@ -256,12 +256,11 @@ void GraphKit::gen_stub(address C_function,
 
   assert (StubRoutines::forward_exception_entry() != nullptr, "must be generated before");
   Node *exc_target = makecon(TypeRawPtr::make( StubRoutines::forward_exception_entry() ));
```

Comment:
`exc_target` is no longer used, so this should probably be removed

Reply:
removed